src/test/scala/uk/gov/nationalarchives/dp/client/DataProcessorTest.scala

```diff
@@ -478,4 +478,54 @@ abstract class DataProcessorTest[F[_]](implicit cme: MonadError[F, Throwable]) e
 
     exception.getMessage should equal(s"Entity not found for id $id")
   }
+
+  "childNodeFromWorkflowInstance" should "return the node requested" in {
+    val input =
+      <WorkflowInstance xmlns="http://workflow.preservica.com">
```

Comment:
Just a suggestion so feel free to ignore but do you need all of the XML elements here when you're only testing for `Id` It'd make the test shorter if you just have
```xml
<WorkflowInstance xmlns="http://workflow.preservica.com">
  <Id>3</Id>
</WorkflowInstance>
```

You could do something similar in the other test as well.